openapiart/bundler.py

```diff
@@ -93,6 +93,7 @@ def bundle(self):
             self._read_file(self._base_dir, self._api_filename)
 
         self._resolve_x_include()
+        self._validate_x_field_pattern()
```

Comment:
should be called inside `_resolve_x_pattern`

also, is there a reason why we pass a value "x-field-pattern" ? wouldn't it just make sense to call this method `_resolve_x_field_pattern` ?

Reply:
done